src/repositories/documents/postgres/documents.ts

```diff
@@ -38,7 +42,14 @@ export default class DocumentsRepo implements DocumentsInterface {
     };
     const { html } = template;
     const templateHtml = `./data/html/${html}`;
-    const compiledHtml = handlebars.compile(fs.readFileSync(templateHtml).toString());
+    const filekey = `concert-templates/${html}`;
+    const downloadedRes = await this.fileManagerRepository.downloadFile(filekey);
+    let compiledHtml;
+    if (downloadedRes.success) {
+      compiledHtml = handlebars.compile(downloadedRes.data.toString());
+    } else {
+      compiledHtml = handlebars.compile(fs.readFileSync(templateHtml).toString());
```

Comment:
add a TODO comment that this needs to be removed once we have the InMemoryFileManager implemented